### PR TITLE
Rc 3 7 2

### PR DESCRIPTION
**Pull request by BFH-ktt1**

### CODE REVIEW

File: src/main/resources/resources/logisim/strings/gui/gui_fr.properties [hunk 1]
```diff
@@ -618,6 +618,7 @@ windowGridBgColor = Couleur de fond si grille :
 windowGridDotColor = Couleur des points d'unitée :
 windowGridZoomedDotColor = Couleur des points de sous unitée :
 windowGridColorsReset = Réinitialiser les couleurs de la grille aux valeurs par défaut de Logisim
+# ==> windowCanvasLocation =
```
Comment (maehne) [inline src/main/resources/resources/logisim/strings/gui/gui_fr.properties:621]: "Emplacement de la surface de dessin :"

File: src/main/resources/resources/logisim/strings/gui/gui_nl.properties [hunk 2]
```diff
@@ -618,6 +618,7 @@ windowGridBgColor = Grid achtergrond kleur
 windowGridDotColor = Grid punt kleur
 windowGridZoomedDotColor = Uitvergrote grid punt kleur
 windowGridColorsReset = Herstel logisims standard grid kleuren
+# ==> windowCanvasLocation =
```
Comment (maehne) [inline src/main/resources/resources/logisim/strings/gui/gui_nl.properties:621]: @BFH-ktt1: Maybe, you could provide a suitable translation here?

**Review by maehne — APPROVED** [review]
Looks good to me. The white space fixes and suggested localisation strings don't need to be forcibly part of this release, but maybe only applied to our `develop` branch.

File: src/main/java/com/cburch/logisim/std/wiring/PinAttributes.java [hunk 3]
```diff
@@ -80,7 +80,7 @@ public boolean isToSave(Attribute<?> attr) {
     if (attr == ATTR_DUMMY) return (V) "nochange";
     return super.getValue(attr);
   }
-
+
```
[the remark this answers is not in the document]
Comment (BFH-ktt1) [reply src/main/java/com/cburch/logisim/std/wiring/PinAttributes.java:83]: Will do, kept into my history to touch in develop

**Pull request merged**